src/details/ArborX_Predicates.hpp

```diff
@@ -81,6 +105,16 @@ KOKKOS_INLINE_FUNCTION int getK(Nearest<Geometry> const &pred)
   return pred._k;
 }
 
+namespace Experimental
+{
+template <typename Geometry>
+KOKKOS_INLINE_FUNCTION OrderedIntersection<Geometry>
+ordered_nearest(Geometry const &geometry)
```

Comment:
It needs to say "intersect"
I am tempted to define a tag `Experimental::Ordered` and define
```
nearest(Experimental::Ordered, Geometry const&)